e2e/command_test.go

```diff
@@ -244,7 +245,8 @@ func TestE2E_ReenableTaskTriggers(t *testing.T) {
 
 	cts, stop := api.StartCTS(t, configPath)
 	t.Cleanup(func() {
-		cleanup()
+		err := cleanup()
+		require.NoError(t, err)
```

Comment:
I'm not so sure we should be failing on the cleanup like this. If there is an issue with deleting a temporary directory, then we'd have to rerun the tests even though the behavior of CTS was passing.

Reply:
Addressed in https://github.com/hashicorp/consul-terraform-sync/pull/653/commits/40cb2fc4f6637d896e608977841e3c4e022645ac